snap/hooks/install

```diff
@@ -10,6 +10,10 @@ export OPENSSL_CONF="/snap/microk8s/current/etc/ssl/openssl.cnf"
 source $SNAP/actions/common/utils.sh
 
 cp -r ${SNAP}/default-args ${SNAP_DATA}/args
```

Comment:
During installation the default arguments are copied under `/var/snap/microk8s/current/args`. You do not need to change anything in the install hook. The logic you have here should go into the configure hook. It is a common pattern so have a look at the helper functions used.

Reply:
moved to configure hook